src/features/profile/myProfile/MyProfile.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import ProfileCard from '../components/MyProfileCard'
+import ProfileRelations from '../components/ProfileRelations'
+import ProfileGeneral from '../components/MyProfileGeneral'
+import PageHeader from '../../../common/components/PageHeader'
+
+const Profile = (): JSX.Element => {
+  const [editMode, setEditMode] = useState(false)
+  const [isEmpty, setEmpty] = useState<boolean>(false)
+  const [tab, setTab] = useState(0)
+
+  const onTabToggle = (index: number) => {
+    setTab(index)
+  }
+
+  return (
+    <div className='mx-auto w-full bg-gray-f8 text-gray-23'>
+      <div className='wrapper pt-0 h-35px flex items-center'>
+        <div className='text-sm text-gray-71'>My account / General</div>
+      </div>
```

Comment:
@reacteer I think this is static. Please use Breadcrumbs component, refer PR#144